qutebrowser/browser/commands.py

```diff
@@ -552,6 +552,17 @@ def _back_forward(self, tab, bg, window, count, forward):
         else:
             widget = self._current_widget()
 
+        if not count and index:
+            try:
+                index = int(index)
+            except ValueError:
+                raise cmdexc.CommandError("Invalid int value {}"
```

Comment:
How about "index must be an int" or something so the user knows which argument they messed up?

Reply:
Okay, although that is the only non-flag argument back and forward take.